scripts/parser/main.go

```diff
@@ -1,117 +0,0 @@
-// (c) 2022 Ava Labs, Inc. All rights reserved.
```

Comment:
Should we remove this entire directory if this file is going to be removed?

Reply:
yes